build.gradle

```diff
@@ -3533,6 +3533,7 @@ project(':connect:runtime') {
     implementation libs.classgraph
     implementation libs.mavenArtifact
     implementation libs.swaggerAnnotations
+    implementation libs.javaxWsRs
```

Comment:
could you please take a look at my suggested approach in the jira? we should NOT add it back to production scope. it is used by swagger scope only